app/views/schools/audits/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<% content_for :page_title, "#{@school.name} Audit - #{@audit.title}" %>
+<% content_for :page_title, "#{@school.name} #{t('schools.audits.show.title')} - #{@audit.title}" %>
```

Comment:
Should this whole line be translated, as the sentence order might be different in Welsh?

Reply:
:+1: have pushed updates to this and the above for actions.  would you be able to give this another quick review before i merge.  thanks :)